tests/components/launch_library/test_config_flow.py

```diff
@@ -0,0 +1,54 @@
+"""Test launch_library config flow."""
+
+from homeassistant import data_entry_flow
+from homeassistant.components.launch_library.const import DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+
+from tests.common import MockConfigEntry
+
+
+async def test_import(hass):
+    """Test entry will be imported."""
+
+    result = await hass.config_entries.flow.async_init(
```

Comment:
Don't we need to patch the setup entry from the integration to prevent outgoing connections?

Reply:
I am novice so excuse my dumb question 😄 

It does not make any API calls in the config flow, only after it have been setup (eg. to update data), as no user input is needed, so what am do I patch? 

Thanks for the help, learning a lot 😃

Reply:
I believe I have figured it out on my own 😉